### PR TITLE
Restricted pages

### PR DESCRIPTION
**Pull request by clarkwinkelmann**

### CODE REVIEW

File: js/src/admin/components/PagesListItem.js [hunk 1]
```diff
@@ -14,9 +15,10 @@ export default class PagesListItem extends Component {
     view() {
         const page = this.props.page;
         const url = app.forum.attribute('baseUrl') + '/p/' + page.id() + '-' + page.slug();
+        const badges = page.badges().toArray();
         return (
             <tr key={page.id()}>
-                <th>{page.title()}</th>
+                <th>{page.title()} {badges.length ? m('ul.badges', listItems(page.badges().toArray())) : null}</th>
```
Comment (dsevillamartin) [inline js/src/admin/components/PagesListItem.js]: I don't think we should be mixing hyperscript & JSX - stick to JSX.

Comment (clarkwinkelmann) [reply js/src/admin/components/PagesListItem.js]: How do I do that in JSX ? :sweat_smile: 

EDIT: oh maybe `(<ul></ul>)` ? I'll test

Comment (dsevillamartin) [reply js/src/admin/components/PagesListItem.js]: nice edits

```suggestion
                <th>{page.title()}
                	{badges.length &&
                		<ul className="badges">
                			{listItems(page.badges().toArray()}
                		</ul>
                	}
                </th>
```

Comment (clarkwinkelmann) [reply js/src/admin/components/PagesListItem.js]: Fixed! thanks :hugs:

**Pull request merged**